inventory_management_system_api/routers/v1/setting.py

```diff
@@ -37,3 +37,16 @@ def update_spares_definition(
         message = "A specified usage status does not exist"
         logger.exception(message)
         raise HTTPException(status_code=status.HTTP_422_UNPROCESSABLE_ENTITY, detail=message) from exc
+
+
+@router.get(
+    path="/spares_definition", summary="Get the definition of a spare", response_description="Spares definition"
+)
+def get_spares_definition(setting_service: SettingServiceDep) -> SparesDefinitionSchema:
+    # pylint: disable=missing-function-docstring
+    logger.info("Getting spares definition")
+
+    spares_definition = setting_service.get_spares_definition()
+    if not spares_definition:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="Spares definition not found.")
+    return spares_definition
```

Comment:
I see this warning in Pycharm `Expected type 'SparesDefinitionSchema', got 'SparesDefinitionOut' instead`.

Reply:
Ah yeah good spot 😄, fastapi handles the conversion the the return type automatically so it didn't break but we always convert them ourselves.